Bugfixes for events feature

rjsughosh opened the pull request:
Description:
Bugfixes:

- Label change in create-service for DynamoDB and SQS

- Limiting Events rate and aws events selection

- displaying selected aws event in metadata

- form reset for events

svidhani commented on builds/jenkins-build-pack-api/Jenkinsfile:
```diff
@@ -520,26 +520,26 @@ def generateSwaggerEnv(env, deploymentNode, apiHostName, config) {
 	sh "sed -i -- 's/\${file(deployment-env.yml):service}/${configLoader.INSTANCE_PREFIX}-${config['domain']}-${config['service']}/g' serverless.yml"
-	sh "sed -i -- 's/{env-stage}/${environment_logical_id}/g' serverless.yml"
+	sh "sed -i -- 's/{inst_stack_prefix}/${configLoader.INSTANCE_PREFIX}/g' serverless.yml"
+  sh "sed -i -- 's/{env-stage}/${environment_logical_id}/g' serverless.yml"
```
indentation here and other places

devsatishm commented:
Updated the base to `events` branch.

devsatishm commented:
Found of couple of issues - 

- UI shouldn't remember the previous selection (after service creation is successful) - seen it with 'Scheduled Events' component - selected a schedule, service creation was successful but when I click on create service button again, previous selection still remains)
- In service overview panel, we are now showing `AWS Events: dynamodb`. We'll need to show both the event source type (DynamoDB, S3, SQS, Kinesis) & actual event source (ARN in this case). Preferred labels - `Event Source` & `Event Source Type`

rjsughosh commented:
> Found of couple of issues -
> 
> * UI shouldn't remember the previous selection (after service creation is successful) - seeing it with 'Scheduled Events' component (Selected a schedule, service creation was successful but when I click on create service button again, previous selection still remains).
> * In service overview panel, we are now showing `AWS Events: dynamodb`. We'll need to show both the event source type (DynamoDB, S3, SQS, Kinesis) & actual event source (ARN in this case). Preferred labels - `Event Source` & `Event Source Type`

Fixed these issues, updated PR
